### ISSUE TITLE
Make summaries more accurately account user ops

### ISSUE THREAD

**Issue by vladsud** [microsoft/FluidFramework#9922]
We use number of ops since last summary as one of the triggers for summary generation:

```ts
    public get opsSinceLastAck(): number {
        return this.heuristicData.lastOpSequenceNumber - this.heuristicData.lastSuccessfulSummary.refSequenceNumber;
    }
```

This counts all ops, system ops and runtime ops.
Runtime ops matter 100%, because they change document state.
Summarizing system ops matters substantially less, as they do not modify document state, thought if we have a lot of them, it will take a bit longer to boot (as it takes some time to apply them, though processing time is much faster for system ops as less code is involved).

The ideal solution is that we use different weights here - 1.0 for runtime ops, and let's say 0.1 for system ops.

Andrei just added tracking this difference for telemetry purposes only: https://github.com/microsoft/FluidFramework/pull/9687/files
It would be great to leverage that.

Can we pass this.opTracker to through layers to SummarizeHeuristicRunner object?
Actually, I think it would be better for RunningSummarizer to control all of that (see handleOp() handler), do this counting directly in this class, and
- Pass extra info to this.heuristicRunner?.run() such that it can do weighted analyzes
- Account these properties in trySummarize() method (and thus not need runtime to care about this tracking).

If we can do that, then I'd either reduce maxOps to 100 (see top of https://github.com/microsoft/FluidFramework/issues/9908 for where this setting exists), or simply use coefficients 1.5 & 0.2 (instead of 1.0 and 0.1) to keep old limits in place, but make system summarize more often with same number of user ops, and less often with same number of system ops.

**Comment by vladsud** [comment]
@NicholasCouri, @andre4i - FYI

**Comment by kian-thompson** [comment]
Closed to track in AB#126
